src/components/NewsList/NewsListItem.js

```diff
@@ -14,11 +14,14 @@ class NewsListItem extends Component {
   }
 
   render() {
-    const { newsItem } = this.props;
+    const { newsItem, style } = this.props;
     const { headline, source, snippet } = newsItem;
 
     return (
-      <TouchableOpacity style={styles.container} onPress={this.handleItemPress}>
+      <TouchableOpacity
+        style={(style, styles.container)}
```

Comment:
How this works, it should be `{[styles.container, style]}`. Array + base style should be before override element